position.go

```diff
@@ -39,7 +39,7 @@ func Place(width, height int, hPos, vPos Position, str string, opts ...Whitespac
 
 // PlaceHorizontal places a string or text block horizontally in an unstyled
 // block of a given width. If the given width is shorter than the max width of
-// the string (measured by it's longest line) this will be a noöp.
+// the string (measured by its longest line) this will be a noop.
```

Comment:
Farewell my diaeresis

Reply:
@meowgorithm don't tell me it was intentional :)

Reply:
@nervo Alas, as a native English speaker, this was my chance to flaunt the only diacritic we (barely) have. It was naïve of me to think spell checkers would coöperate accordingly.

:D

Reply:
Now i feel both sad and ashamed...
Here is a lil' duck for you, may its cuteness fill your heart
![](https://static.vecteezy.com/ti/photos-gratuite/p1/2325235-petit-mignon-caneton-en-plein-air-photo.JPG)